week1/homework/Program.cs

```diff
@@ -0,0 +1,84 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+// String Manipulation
+
+app.MapGet("/reverse", (string input) => {
+    char[] charArray = input.ToCharArray();
+     int left = 0;
+        int right = charArray.Length - 1;
+
+        while (left < right)
+        {
+            char temp = charArray[left];
+            charArray[left] = charArray[right];
+            charArray[right] = temp;
+
+            left++;
+            right--;
+        }
```

Comment:
this is a good implementation 👍  
You can also try to use the inbuild methods for arrays/list. Maybe you can find a much easier solution? 😊